docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1911 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    T = ts.copy()
</pre><p>So, just replace this line with:</p><pre class="ql-syntax" spellcheck="false">T, M_T, Σ_T = core.preprocess(T, m)
</pre><p> </p><p>This is actually a really good sign because we are getting more and more lines of code and it is becoming easier to read/maintain!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Thanks for pointing this out!! Will be fixed next time.